src/backend/tools/index.ts

```diff
@@ -835,3 +840,182 @@ export async function runWineCommandOnGame(
     startFolder
   })
 }
+
+/**
+ * Download and Install the Windows version of Steam on a Wine Prefix on macOS only
+ */
+export const SteamWindows = {
+  downloadSteam: async () => {
+    const steamURL =
+      'https://drive.usercontent.google.com/download?id=1MXowtPBihbljXFs_5cM1AfyOjnC3raaT&export=download&authuser=0&confirm=t&uuid=e159346a-5836-4f7d-8aed-af3d8340fef7&at=AENtkXa2d83g4dE_UZYQKKLoEUje%3A1732794843031'
```

Comment:
this looks.... weird

we'll have to be really clear to the user why are we getting a steam zip file from a random google drive account

why would they trust that we are installing something safe if they can't even see where is this coming from?

I understand it's because of the thing you mentioned about steam not supporting windows 7 and GPTK, but we need to be really clear and let the user know what's going on before they confirm anything

Reply:
Other than this looking sketchy, it also won't work since GDrive makes a file unavailable if it's downloaded too frequently (which I assume will happen sometimes). Can we use an `archive.org` link for this?

Reply:
Yeah, I can upload it to archive.org